src/main/java/synapseawsconsolelogin/Auth.java

```diff
@@ -271,6 +274,9 @@ public AssumeRoleRequest createAssumeRoleRequest(Claims claims, String roleArn,
 			}
 		}
 		
+		// this is the fix for SC-178
```

Comment:
could please provide a few additional comments to explain why we need this tag?  The issue is too overwhelming to read thru at this point.

Reply:
Please see the updated PR.